app/app.go

```diff
@@ -165,7 +165,7 @@ func getHealthVal(jobsDB jobsdb.JobsDB) (bool, string) {
 
 	appTypeStr := strings.ToUpper(config.GetString("APP_TYPE", EMBEDDED))
 	return healthy, fmt.Sprintf(
-		`{"appType":"%s","server":"UP","db":"%s","acceptingEvents":"TRUE","mode":"%s",`+
+		`{"appType":"%s","server":"UP","db":"%s","acceptingEvents":"TRUE","routingEvents":"TRUE","mode":"%s",`+
```

Comment:
I guess we wanted to remove mode in place we accidentally removed  `routingEvents `
```suggestion
		`{"appType":"%s","server":"UP","db":"%s","acceptingEvents":"TRUE","routingEvents":"%s",`+
```